packages/wallets/src/components/Base/ModalStepWrapper/ModalStepWrapper.scss

```diff
@@ -3,13 +3,17 @@
     border-radius: 0.8rem;
 
     @include mobile {
-        height: calc(100vh - 4rem);
+        height: calc(100svh - 4rem);
         width: 100vw;
         margin: 0;
         border-radius: 0;
         display: grid;
         grid-template-rows: 6rem auto;
 
+        &--hide-deriv-app-header {
+            height: 100svh;
```

Comment:
[Question]: Is this supported by the old browsers?

Reply:
Actually no 👀 
<img width="1475" alt="Screenshot 2023-11-20 at 1 58 40 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/cf3afa5d-9cca-477e-87f5-7a715cb5988a">
I saw some cases where there are issues in mobile Safari, I think I'll revert this for now

Reply:
@matin-deriv @adrienne-deriv we already used svh for .wallets-app and for .wallets-cashier so we need to change it too in the case we don't want to use these new css properties. Earlier we used Div100vh library to solve this issue so if we don't want to use these new properties then I guess we have only one solution for mobile browsers